packages/sanity/src/core/form/studio/tree-editing/components/TreeEditingDialog.tsx

```diff
@@ -219,6 +225,7 @@ export function TreeEditingDialog(props: TreeEditingDialogProps): JSX.Element |
           <Card borderTop>
             <Flex align="center" justify="flex-end" paddingX={3} paddingY={2} sizing="border">
               <Button
+                size="large"
```

Comment:
The large `ui-components` button equals to the default size of the button imported from `@sanity/ui`